arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -0,0 +1,75 @@
+# Running an Archive Node
+
+Note: If you’re interested in accessing the Arbitrum network but you don’t want to setup your own node, see our [Node Providers](./node-providers.mdx) to get RPC access to fully-managed nodes hosted by a third party provider.
+
```

Comment:
We should add a note here says this docs is for arbitrum one (Only Arb1 need run both nitro and classic), if they want to run archive node for arb-goerli or nova, they can just see [running-a-node.mdx](./running-a-node.mdx)